tests/5.2/scope/test_scope_firstprivate_construct_mod.c

```diff
@@ -0,0 +1,41 @@
+//--------------- test_scope_firstprivate_construct_mod.c -----------------------------//
```

Comment:
im a bit confused abt the test name, what is construct_mod?

Reply:
The test name this test refers to is test_scope_firstprivate_construct and I added _mod to indicate that the new test is a modification of that test file. I followed the same naming style on the training branch, which was for my first test, so I can change as needed.